.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: CI
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  molecule:
+    runs-on: ubuntu-latest
```

Comment:
@sabre1041 I assume ubuntu is only to run molecule, right? Even then, it would be nice to be able to run on Fedora/Centos/Scientific Linux over Ubuntu.

Reply:
@rpelisse the designation of ubuntu is scoped specifically where you run molecule. the tests execute within the same `centos:systemd` container